packages/ui-kit-stage/HeaderBar.tsx

```diff
@@ -13,7 +13,12 @@ const headerBarStyles: object = {
   flexShrink: 1,
   flexGrow: 0,
   flexBasis: "auto",
-  zIndex: 999
+  zIndex: 999,
+  display: "flex",
+  paddingLeft: "20px",
```

Comment:
this should be handled by the children IMHO 🤔

Reply:
Can you clarify? Are you talking about just the padding?

Reply:
@judithpatudith I believe this should be in the parent instead of the children that way creating this wrapper where whenever we have a new child the component will be positioned at the same place we expect.

If we add to the child whenever we replace/ or add elements to the edges we will need to re-apply the padding.

Reply:
I asked @tnguyen0 to try using the cnvs grid system.

Reply:
Took a look at cnvs, didn't appear applicable to me.

Reply:
After looking at this again, I think it has to be changed… on my machine, the logo is centered instead of left-aligned. 
<img width="1106" alt="screen shot 2018-07-17 at 13 45 59" src="https://user-images.githubusercontent.com/530632/42815474-c1813c7c-89c7-11e8-803f-78b8ba94807d.png">

Reply:
Refer to testing steps. `<span>toggle component here</span>` needs to be commented out.

Reply:
Yeah, the fix would be to put the toggle component in a wrapper with the logo. This can be fixed in the UI issue and truncation ticket.

Reply:
just dont do `justifyContent: "space-between"` :D 

Can we please remove the styles from Headerbar and fix that in DCOS-38944? 🙇